spec/requests/editions_spec.rb

```diff
@@ -10,5 +10,28 @@
 
       expect(response.body).to include(I18n.t!("documents.edit.title", title: edition.title))
     end
+
+    context "when the edition is in history mode" do
+      let(:edition) { create(:edition, :published, :political, :past_government) }
+
+      it "lets users holding the manage_live_history_mode permisssion create a new edition" do
+        user = create(:user, manage_live_history_mode: true)
+        login_as(user)
+        stub_publishing_api_put_content(edition.content_id, {})
+        post create_edition_path(edition.document)
+        follow_redirect!
+
+        expect(response.body).to include(I18n.t!("documents.edit.title", title: edition.title))
```

Comment:
This has the same issue of not really testing what it says it does.

Reply:
Updated the test!